0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -71,11 +71,14 @@ For more details on the flow, see the [Interledger Protocol specification](../00
 
 <span class="show alert alert-info">**Note:** Interledger only supports Universal mode as described in the whitepaper. Atomic mode can be used by adjacent subsets of participants in an Interledger payment if desired, but this is not part of the standard.</span>
 
-### Error Recovery and Connector Risk
+### Connector Risk and Mitigation
 
-An ILP execution chain can be interrupted if a connector fails to deliver the condition fulfillment. In this case, the sender thinks the transaction failed and retries it using the same condition as the original transaction. If the retry goes through the same connector that failed the first time, that connector can claim the incoming funds without sending a second outgoing transfer.
+Interledger protects senders from risk but connectors accept some risk in exchange for the revenue they generate from facilitating payments. In the Interledger payment flow, connectors' outgoing transfers are executed before their incoming transfers. Once each connector is notified that the outgoing transfer has been executed, they have a window of time to deliver the fulfillment and execute the incoming transfer. Connectors that fail to deliver the fulfillment in time may lose money.
+
+If some transfers in an Interledger payment are executed and others expire, the receiver will think the payment was completed but the sender may think the whole payment failed. Senders may retry payments that expire and they should use the same condition as the original payment. If the second attempt takes the same route, the connector that failed the first time can complete the payment by submitting the fulfillment without sending another outgoing transfer. Connectors should prefer senders (or aggregators of senders such as wallet services) and connectors that retry payments through the same route, because they increase the likelihood of completing such payments. Connectors may incentivize this behavior by offering better rates to parties known to retry failed payments through the same route.
```

Comment:
add a sentence explaining that the sender could also just ask the receiver out-of-band whether the money was delivered. in itself, buying this Boolean piece of information from the connector may not always be worth it for the sender, unless the transfer amount was smaller than the incentive, and out-of-band communication with the receiver is hard.